src/providers/twitch/TwitchChannel.cpp

```diff
@@ -1253,7 +1253,11 @@ void TwitchChannel::refreshPubSub()
     getApp()->twitch->pubsub->setAccount(currentAccount);
 
     getApp()->twitch->pubsub->listenToChannelModerationActions(roomId);
-    getApp()->twitch->pubsub->listenToAutomod(roomId);
+    if (this->hasModRights())
+    {
+        getApp()->twitch->pubsub->listenToAutomod(roomId);
+        getApp()->twitch->pubsub->listenToLowTrustUsers(roomId);
+    }
```

Comment:
Could you make sure to try the low trust user topic out again after this change?
I've tested it & it seems to work, but I haven't given it super much testing